<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Product</class>
 <widget class="QDialog" name="Product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="Pricelabel">
       <property name="text">
        <string>Цена </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Priceline">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Weightlabel">
       <property name="text">
        <string>Вес</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Weightline">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Quantitylabel">
       <property name="text">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Quantityline">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Sizelabel">
       <property name="text">
        <string>Размер</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="SizeBox">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Platformlabel">
       <property name="text">
        <string>Платформа</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="PlatformBox">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wildberries</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ozon</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Articlelabel">
       <property name="text">
        <string>Артикул</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="Articleline">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Clientlabel">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Клиент</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Clientline">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="DeliveryDatelabel">
       <property name="text">
        <string>Дата доставки</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCalendarWidget" name="DeliveryDateWidget">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="gridVisible">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QCalendarWidget::SelectionMode::SingleSelection</enum>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::HorizontalHeaderFormat::ShortDayNames</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_btn">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancal_btn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
